2021-08-29 10:41:54,279 INFO    MainThread:44995 [wandb_setup.py:_flush():69] setting env: {}
2021-08-29 10:41:54,279 INFO    MainThread:44995 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-29 10:41:54,279 INFO    MainThread:44995 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/code/image-classification-level1-22/T2037/wandb/run-20210829_104154-1y1ykegk/logs/debug.log
2021-08-29 10:41:54,279 INFO    MainThread:44995 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/code/image-classification-level1-22/T2037/wandb/run-20210829_104154-1y1ykegk/logs/debug-internal.log
2021-08-29 10:41:54,279 INFO    MainThread:44995 [wandb_init.py:init():375] calling init triggers
2021-08-29 10:41:54,279 INFO    MainThread:44995 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-08-29 10:41:54,279 INFO    MainThread:44995 [wandb_init.py:init():424] starting backend
2021-08-29 10:41:54,279 INFO    MainThread:44995 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-29 10:41:54,291 INFO    MainThread:44995 [backend.py:ensure_launched():135] starting backend process...
2021-08-29 10:41:54,298 INFO    MainThread:44995 [backend.py:ensure_launched():139] started backend process with pid: 45018
2021-08-29 10:41:54,299 INFO    MainThread:44995 [wandb_init.py:init():429] backend started and connected
2021-08-29 10:41:54,300 INFO    MainThread:44995 [wandb_init.py:init():477] updated telemetry
2021-08-29 10:41:54,301 INFO    MainThread:44995 [wandb_init.py:init():500] communicating current version
2021-08-29 10:41:55,035 INFO    MainThread:44995 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-29 10:41:55,035 INFO    MainThread:44995 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-29 10:41:55,254 INFO    MainThread:44995 [wandb_init.py:init():540] starting run threads in backend
2021-08-29 10:42:00,259 INFO    MainThread:44995 [wandb_run.py:_console_start():1584] atexit reg
2021-08-29 10:42:00,259 INFO    MainThread:44995 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-08-29 10:42:00,260 INFO    MainThread:44995 [wandb_run.py:_redirect():1463] Redirecting console.
2021-08-29 10:42:00,263 INFO    MainThread:44995 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-29 10:42:00,263 INFO    MainThread:44995 [wandb_init.py:init():565] run started, returning control to user process
2021-08-29 10:42:03,414 INFO    MainThread:44995 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 1
2021-08-29 10:42:03,414 INFO    MainThread:44995 [wandb_run.py:_restore():1526] restore
2021-08-29 10:42:06,309 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:42:06,600 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:42:07,279 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10920
}

2021-08-29 10:42:07,381 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10920
}

2021-08-29 10:42:07,489 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10920
}

2021-08-29 10:42:07,590 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10920
}

2021-08-29 10:42:07,692 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:42:07,794 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:42:07,895 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:42:07,997 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:42:08,099 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:42:08,817 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:42:08,920 INFO    MainThread:44995 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10920
  total_bytes: 10920
}

2021-08-29 10:42:10,032 INFO    MainThread:44995 [wandb_run.py:_show_files():1898] logging synced files
